src/ClockKit/CLKComplication.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Runtime.InteropServices;
+using Foundation;
+using CoreFoundation;
+using ObjCRuntime;
+
+namespace ClockKit {
+	public partial class CLKComplication {
+			[Watch (7,0)]
+			[DllImport (Constants.ClockKitLibrary)]
+			static extern IntPtr CLKAllComplicationFamilies ();
+
+			public static NSNumber[] GetAllComplicationFamilies ()
```

Comment:
`NSNumber` is not a very nice type to expose and you already did a better job elsewhere in the bindings ;-)

```
		[Export ("supportedFamilies")]
 		[BindAs ( typeof (CLKComplicationFamily []))]
 		NSNumber[] SupportedFamilies { get; }
```